doc/user/content/sql/functions/filters.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Aggregate Function Filters"
+description: "If FILTER is specified, then only the input rows for which the filter_clause evaluates to true are fed to the aggregate function; other rows are discarded"
+menu:
+main:
+parent: 'sql-functions'
+---
+
+If `FILTER` is specified, then only the input rows for which the `filter_clause` evaluates to true are fed to the [aggregate function](../#aggregate-func); other rows are discarded.
```

Comment:
You can use a `FILTER` clause on an aggregate function to specify which rows are sent to an aggregate function. Rows for which the `filter_clause` evaluates to false are discarded.